guides/_subsections/common/select-target-application.md

```diff
@@ -5,7 +5,7 @@
   {{ include.select-target-application-additional-content }}
 {% endcapture %}
 {% include howto_step.html
-  summary="Select the target application"
+  summary="Select the <i>Target Application</i>"
```

Comment:
Why is target application italicized?

Reply:
the idea is to italicize `select/click` buttons or related items

Reply:
ok I thought it was just the actual text on a button that would be italicized. This is helpful, thanks.